client/common/sass/_responsive.sass

```diff
@@ -17,6 +17,10 @@ $desktop-min: $tablet-max + 1px
 	@media(max-width: $desktop-min)
 		@content
 
+=mobile-down
```

Comment:
I don't have any strong opinion on this. I also saw this in the documentation, but since it was already being used with `=` and `+` notation, I just kind of rolled with it.